### PR TITLE
Fixes #169: add support for wirepath bullet camera

### LINKED ISSUE #169: Add support for WirePath WPS-750-BUL Camera
MAC: 

000F0D
D46A91

RTSP: 

main stream: `rtps://{{username}}:{{password}}@{{ip_address}}:554/`
sub stream: `rtsp://{{username}}:{{password}}@{{ip_address}}:554/v2`

### PR DESCRIPTION
**Pull request by jhallard**

### CODE REVIEW

**Comment by jhallard** [comment]
@jcmaslan the RTSP paths in this case are supposed to be 

1. `/` - for main stream
2. `/v2` - for sub stream

so I made the streams `;v2`, expecting this to be parsed to `""` and `"v2"`. Does this make sense?

**Comment by jcmaslan** [comment]
yes, I'm not sure that I've tested empty values but we'll soon find out!

File: cameras/paths.csv [hunk 1]
```diff
@@ -41,6 +41,7 @@ Vivotek,camera,CC8130,(00:02:D1),rtsp://{{username}}:{{password}}@{{ip_address}}
...
 IC Realtime,nvr,NVR,(52:54:4C),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel}}&subtype={{stream}},H.264,554,0;1,01;02;03;04;05;06;07;08;09;10;11;12;13;14;15;16,High;Low,Channel {{channel}},1,0,admin,1234,,http://icrealtime.com/docs/product-manual/Man_DVRFLEX4E.pdf
 Luma,nvr,Lum-500/501 Series,(00:0F:0D),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}{{channel}}/Streaming/channels/{{channel}}{{stream}},H.264,554,01;02,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,main stream;sub stream,Channel {{channel}},01,02,admin,admin,,https://www.snapav.com/wcsstore/ExtendedSitesCatalogAssetStore/attachments/documents/Surveillance/ManualsAndGuides/Luma-500%20Technical%20Reference%20Manual.pdf
+Wirepath,camera,IP Camera,(00:0F:0D|D4:6A:91),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}},H.264,554,;v2,,Main Stream;Sub Stream,,,v2,admin,admin,,https://www.snapav.com/wcsstore/ExtendedSitesCatalogAssetStore/attachments/documents/Surveillance/ManualsAndGuides/WPS-750-BUL-IP-GR_Manual.pdf
```
Comment (jcmaslan) [inline cameras/paths.csv]: are 44 and 45 intentionally overlapping even though both are cameras with the same OUI?

**Review by jcmaslan — APPROVED** [review]
assuming row 44/45 overlap is intended.

File: cameras/paths.csv (hunk 1 above)
Comment (jhallard) [reply cameras/paths.csv]: 45 was preexisting but for whatever reason that RTSP url was not working for the user that I was supporting, only the new URL was working. I couldn't remember if we had an old user with the HTTP URL (line 45), so I left both assuming that the firmware would try both and choose the one that works.

**Pull request merged**